### PR TITLE
IOBinding docs

### PR DESCRIPTION
**Pull request by duli2012**
**Description**: Describe your changes.
Docs for IOBinding python API

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: docs/python/api_summary.rst [hunk 1]
```diff
@@ -9,6 +9,59 @@ in *ONNX Runtime*.
...
+
+Scenario 2:
+
+The input data is on a device, users direclty use the input. The output data is on CPU.
+
+.. code-block:: python
+
+	session = onnxruntime.InferenceSession('model.onnx')
+	io_binding = session.io_binding()
+	io_binding.bind_input('input', X.device.type, 0, np.float32, list(X.size()), X.data_ptr())
```
Comment (tianleiwu) [inline docs/python/api_summary.rst:48]: Add description for each parameter of bind_input

File: docs/python/api_summary.rst [hunk 2]
```diff
@@ -9,6 +9,59 @@ in *ONNX Runtime*.
...
+
+.. code-block:: python
+
+	#X is numpy array on cpu 
+	session = onnxruntime.InferenceSession('model.onnx')
+	io_binding = session.io_binding()
+	io_binding.bind_cpu_input('input', X)
+	io_binding.bind_output('output')
+	session.run_with_iobinding(io_binding)
+	Y = io_binding.get_outputs()[0]
```
Comment (hariharans29) [inline docs/python/api_summary.rst:38]: get_outputs() -> copy_outputs_to_cpu() everywhere in the file

Comment (duli2012) [reply docs/python/api_summary.rst]: done

File: docs/python/api_summary.rst (hunk 1 above)
Comment (duli2012) [reply docs/python/api_summary.rst]: done

**Pull request merged**